tests/support/server.rs

```diff
@@ -23,6 +23,7 @@ pub struct Txn {
     pub read_timeout: Option<Duration>,
     pub response_timeout: Option<Duration>,
     pub write_timeout: Option<Duration>,
+    pub chunk_size: Option<usize>,
```

Comment:
I must've been doing something else wrong here before, because changing this back to emit the body in chunks without explicit chunked encoding is giving us individual chunks and hitting the right `WouldBlock`s...